tools/bin/cldev

```diff
@@ -1,57 +1,17 @@
 #!/bin/bash
 
-# Runs a Chainlink (CL) node preconfigured to communicate with gethnet.
-# Steps:
-# 1. ./tools/bin/gethnet
-# 2. yarn install
-# 3. yarn workspace chainlink run truffle:migrate:cldev
-# 4. ./tools/bin/cldev
-
 set -e
-
-GIT_ROOT=`git rev-parse --show-toplevel`
-PROJECT_ROOT=${TRAVIS_BUILD_DIR:-$GIT_ROOT}
-LDFLAGS="`$PROJECT_ROOT/tools/bin/ldflags`"
-
-pushd $PROJECT_ROOT >/dev/null
-source tools/bin/clenv
-export ROOT=$PROJECT_ROOT/tools/clroot
-DEFAULT_KEY_PATH=$ROOT/keys/UTC--2017-01-05T20-42-24.637Z--9ca9d2d5e04012c9ed24c0e513c9bfaa4a2dd77f
-
-mainexec() {
-  mkdir -p tmp
-  rm -f core/services/*-packr.go
-  go build -o tmp/cldevbuild -ldflags "$LDFLAGS" ./core/main.go
-  tmp/cldevbuild $@
-}
-
-if [ "$#" == 0 ]; then
```

Comment:
How come this takes out the front end?

Reply:
It's been superseded by `cldo`, which runs `operator-ui` in its own container in dev-mode (any changes made to its codebase are recompiled and hot-reloaded`